lib/Sema/TypeCheckType.cpp

```diff
@@ -3676,6 +3690,35 @@ TypeResolver::resolveCompileTimeConstTypeRepr(CompileTimeConstTypeRepr *repr,
   return resolveType(repr->getBase(), options);
 }
 
+NeverNullType
+TypeResolver::resolveDistributedKnownToBeLocal(DistributedKnownToBeLocalTypeRepr *repr,
+                                               TypeResolutionOptions options) {
+  Type type = resolveType(repr->getBase(), options);
+
+  // '_local' is only value for non-EnumCaseDecl parameters.
+  if (!options.is(TypeResolverContext::FunctionInput) ||
+      options.hasBase(TypeResolverContext::EnumElementDecl)) {
+    diagnoseInvalid(
+        repr, repr->getSpecifierLoc(), diag::attr_only_on_parameters,
+        "_local");
+    return ErrorType::get(getASTContext());
+  }
+
+  // TODO(distributed): more diagnosis here, prevent from use in props, enums etc
+
+  // isolated parameters must be of 'distributed actor' type
+  if (!type->hasTypeParameter() &&
```

Comment:
Presumably it's okay for it to have a type parameter, but then the archetype would have to be a distributed actor (i.e., conform to the `DistributedActor` protocol).

Reply:
Good point, I think that's right... will think about it and try

Reply:
Ok awesome :)